packages/wallet-connectors/CHANGELOG.md

```diff
@@ -16,6 +16,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   `WalletConnect`: Fix schema object format conversion in `signAndSendTransaction` in request payloads.
 -   `WalletConnect`: Use standard string identifiers for transaction type in request payload.
 
+### Removed
+
```

Comment:
Please update the `READMe.md` file in `react-components` package. The file mentions the `jsonRpcUrl`.